<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibrateMag3110</class>
 <widget class="QWidget" name="calibrateMag3110">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>346</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>82</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_start">
      <property name="minimumSize">
       <size>
        <width>108</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_stop">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>83</y>
     <width>290</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>MXMax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="m_MXMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>AXMax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="m_AXMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>MXMin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="m_MXMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>AXMin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="m_AXMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>MYMax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="m_MYMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>AYMax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="m_AYMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>MYMin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="m_MYMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>AYMin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="m_AYMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>MZMax</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="m_MZMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>AZMax</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="m_AZMax">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>MZMin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="m_MZMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>AZMin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="m_AZMin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
